packages/mdc-textfield/README.md

```diff
@@ -176,24 +185,55 @@ UX for client-side form field validation.
 </p>
 ```
 
-### Multi-line - With Javascript
+### Leading and Trailing Icons
+Leading and trailing icons can be added to MDC Textfields as visual indicators
+as well as interaction targets. To do so, add the relevant classnames to the root element, add
+an `i` element with your preferred icon, and give it either a `mdc-textfield__leading-icon` or
+`mdc-textfield__trailing-icon` class depending on which you are looking to achieve.
+
+#### Leading:
+```html
+<div class="mdc-textfield mdc-textfield--box mdc-textfield--with-leading-icon">
+  <i class="material-icons mdc-textfield__leading-icon" tabindex="0">event</i>
+  <input type="text" id="my-input" class="mdc-textfield__input">
+  <label for="my-input" class="mdc-textfield__label">Your Name</label>
+  <div class="mdc-textfield__bottom-line"></div>
+</div>
+```
+
+#### Trailing:
+```html
+<div class="mdc-textfield mdc-textfield--box mdc-textfield--with-trailing-icon">
+  <i class="material-icons mdc-textfield__trailing-icon" tabindex="0">event</i>
+  <input type="text" id="my-input" class="mdc-textfield__input">
+  <label for="my-input" class="mdc-textfield__label">Your Name</label>
+  <div class="mdc-textfield__bottom-line"></div>
+</div>
+```
+
+>**NOTE:** if you would like to display un-clickable icons, simply remove `tabindex="0"`,
+and the css will ensure the cursor is set to default, and that actioning on an icon doesn't
+do anything unexpected.
+
+
+### Textarea
 
 ```html
-<div class="mdc-textfield mdc-textfield--multiline">
-  <textarea id="multi-line" class="mdc-textfield__input" rows="8" cols="40"></textarea>
-  <label for="multi-line" class="mdc-textfield__label">Multi-line Label</label>
+<div class="mdc-textfield mdc-textfield--textarea">
+  <textarea id="textarea" class="mdc-textfield__input" rows="8" cols="40"></textarea>
+  <label for="textarea" class="mdc-textfield__label">Textarea Label</label>
 </div>
 ```
 
-### Multi-line - Gracefully Degraded
+### Textarea - CSS Only
 
 ```html
-<label for="css-only-multiline">Multi-line label: </label>
-<div class="mdc-textfield mdc-textfield--multiline">
-  <textarea class="mdc-textfield__input"
-            id="css-only-multiline"
-            rows="8" cols="40"
-            placeholder="Tell the world something about yourself!"></textarea>
+<div class="mdc-textfield mdc-textfield--textarea">
+  <textarea id="textarea-css-only"
```

Comment:
Is there intentionally no `label` element here? I see label elements in our demos, inside the containing `div`, and in the next textarea example in the readme.

Reply:
In CSS only, we don't use a label, we use placeholder text.